src/bin/tikv-server.rs

```diff
@@ -65,7 +65,7 @@ fn print_usage(program: &str, opts: Options) {
 
 fn get_flag_string(matches: &Matches, name: &str) -> Option<String> {
     let s = matches.opt_str(name);
-    info!("flag {}: {:?}", name, s);
+    debug!("flag {}: {:?}", name, s);
```

Comment:
I prefer outputting the command flags here with INFO level, so we can know what really user passes.  

/cc @ngaut